packages/eslint-plugin/tests/docs.test.ts

```diff
@@ -83,11 +83,11 @@ function renderLintResults(code: string, errors: Linter.LintMessage[]): string {
             ? Math.max(1, endColumn - startColumn)
             : line.length - startColumn,
         );
-        const squiggleWithIndent = ' '.repeat(startColumn) + squiggle + ' ';
+        const squiggleWithIndent = `${' '.repeat(startColumn) + squiggle} `;
```

Comment:
This is one of those cases that I think benefits from having the interpolated expression be assigned a variable

Reply:
Here `repeat` and `squiggle` should be two interpolations.